vuepress/docs/next/tutorials/create/mfe/angular.md

```diff
@@ -74,245 +88,172 @@ This is the expected output:
-3. Load the micro frontend from an API.
-:::
-
-### Add Widget
-
-1. Go to `Components > Micro frontends & Widgets` in the Entando App Builder.
-2. Click `Add` at the lower right.
-
-![New widget screen](./img/new-widget-screen.png)
-
-3. Enter the following:
-- `Code: angular_widget` → note: dashes are not allowed
-- `Title: Angular Widget` → for both English and Italian languages
-- `Group: Free Access`
-- `Custom UI:`
-
-``` ftl
-<#assign wp=JspTaglibs[ "/aps-core"]>
-<script async src="<@wp.resourceURL />angular-widget/main-es2015.js"></script>
-<script async src="<@wp.resourceURL />angular-widget/polyfills-es2015.js"></script>
-<script async src="<@wp.resourceURL />angular-widget/runtime-es2015.js"></script>
-
-<angular-widget />
 ```
+2. Add a thumbnail JPEG or PNG file to the root bundle folder. It must be named thumbnail and not exceed 100kB, e.g. thumbnail.png
 
-4. Click `Save`.
-
-::: tip
-`<#assign wp=JspTaglibs[ "/aps-core"]>` gives you access to the `@wp` object where you can use environment variables like `resourceURL`.
-:::
-
-### See It in Action
-
-Let's see the Angular micro frontend in action on our page.
-
-#### Add Page
-
-::: warning Note
-If you've already configured your home page:
-
-<ol type="i">
-  <li>Next to the <b>Home</b> folder, under <b>Actions</b>, click <b>Configure</b>.</li>
-  <li>Skip to the <b>Add Widget</b> section.</li>
-</ol>
-:::
-
-Let's add our widget to the `Home` page.
-
-1. Go to `Pages` → `Management`
-
-2. Next to the `Home` folder, under `Actions`, click `Edit`.
-
-3. Next to `Page Template` select `Service Page`.
+3. Build the components
+```sh
+ent bundle build --all
+```
+4. Generate the Docker image for the bundle 
+```sh
+ent bundle pack
+```
+5. Publish the image
+```sh
+ent bundle publish 
+```
+6. Deploy the bundle to the Hub
+```sh
+ent bundle deploy 
+```
+7. Log into the App Builder and navigate to the Hub. Click `DEPLOY` for `YOUR-BUNDLE`. Click `install` in the pop-up window. 
 
-4. Click `Save and Configure`.
+### Add the Bundle to a Page
+Let's see the Angular micro frontend in action on a page.
 
-#### Add Widget
+1. In the App Builder, go to Page → Management
 
-1. In the Search field in right-hand sidebar, enter `Angular Widget`.
+2. Choose an existing page (or [create a new one](../../compose/page-management.md#create-a-page)). Select Design from its Actions.
 
-2. Drag and drop `Angular Widget` into the `Sample Frame` in the main body of the page.
+3. In the Search field in the right sidebar, enter `YOUR-BUNDLE` name to find your widget. It can also be found under User in the same sidebar.
 
-3. Click `Publish`.
+4. Drag and drop `YOUR-BUNDLE` into the `Sample Frame` in the main body of the page.
+5. Click `Publish`
 
-4. At the upper right, click `Go to Homepage`.
+6. Click on `View Published Page`
 
 ![Angular Micro Frontend](./img/angular-micro-frontend.png)
 
 ::: tip Congratulations!
 You now have an Angular micro frontend running in Entando.
 :::
+
+
```

Comment:
Maybe this is a good time to add a couple CTAs? I was thinking links to Configuration, Communication, and the JHipster tutorial.